corehq/toggles/__init__.py

```diff
@@ -1591,14 +1591,6 @@ def _commtrackify(domain_name, toggle_is_enabled):
     ),
 )
 
-DD_CASE_DATA = StaticToggle(
```

Comment:
Will this automatically remove the FF from the projects that have it set?

Reply:
The enabled items (usernames, in this case) are all stored on a single document (check `raw_doc` for the ID `hqFeatureToggle-dd_case_data`).  Deleting this block of code doesn't remove that document, it'll stay in the DB.

Reply:
huh yeah, I suppose that is a risk - I hadn't thought of that.  I think this is pretty standard for getting rid of feature flags though, so I'm not especially concerned.